Sources/PackageSigning/SignatureProvider.swift

```diff
@@ -232,34 +249,66 @@ struct CMSSignatureProvider: SignatureProviderProtocol {
         }
     }
 
+    private func isValidSignature(
```

Comment:
I think it's better not to do this refactoring and just add `extractSigningEntity` for now.

Reply:
Removed.